library/include/hiptensor/internal/type_traits.hpp

```diff
@@ -270,6 +271,68 @@ namespace std
         hiptensor::detail::Fp16Bits eps(static_cast<uint16_t>(0x7FC0));
         return eps.b16;
     }
+
+    ///////////////////////////////////////////////////////////
```

Comment:
This seems to be new code? Do we need this?

Reply:
you're right, this has sneaked back with the rebase. i have removed it